### DISCUSSION TITLE
Support Google Wallet API

### DISCUSSION THREAD

**Discussion by kogai** [Byron/google-apis-rs#367] [Q&A] [answered]
Hey, thanks for the great library!

I have an API that would like to add supported APIs of google-apis-rs, so please listen.

It is Google Wallet API, which offers to operate loyalty cards or airplane tickets or else in the Google Wallet app.
Since this API has Discovery Document [here](https://walletobjects.googleapis.com/$discovery/rest?version=v1), but Google Discovery Service hasn't shown it in the list of APIs, as and consequence, the API has also not been included in google-apis-rs.

Could you please consider adding this API to the crates under this repository?

FYI, I got to be able to generate a client by trying to put the Discovery Document URL and other information into apis.json in the repository I forked.
https://github.com/kogai/google-apis-rs/compare/0998b2810e91ab7b76847c859f339ed14e57d014..b9bd9f385af94048547628d9f7d678e4a57759d4

**Comment by Byron** [comment, marked as answer]
Indeed, the top-level discovery document is a copy of an older request made against the google discovery endpoint, and won't contain the latest APIs for that reason.

The fork posted here seems like it could be a PR that can be merged, and I am encouraging you to bring the Google Wallet API to the parent repository that way.

Thanks a lot!

**Comment by kogai** [comment]
Sounds great!
I'll PR right away 👍
